grid/recruitment/list-candidates/profile.supr

```diff
@@ -0,0 +1,209 @@
+"""
+List candidates
+List candidates that applied to job positions of your company.
+"""
+name = "recruitment/list-candidates"
+version = "1.0.0"
+
+"""
+List candidates
+Returns paginated list of candidates.
+"""
+usecase ListCandidates safe {
+ input {
+    """
+    Job ID
+    Uniquie identifier of the job the candidate is applying for.
+    """
+    jobId! string!
+    """
+    Page
+    Pagination cursor (obtained from `previousPage` or `nextPage`).
```

Comment:
Add hint to the Pagination model later. (I was confused where is it coming from)

Reply:
Fixed. Thanks.